### ISSUE TITLE
ImageSearch creator problem

### ISSUE THREAD

**Issue by dibolek** [Pulover/PuloversMacroCreator#100]
Image Search creator with checked **"Adjust coordinates to the center of the image"** and set number of color **"variations"** is passing full filename to the _**CenterImgSrchCoords()**_ function including variations number. So coordinates cannot be updated in this function because of wrong filename passed.

Generated code example:
```
ImageSearch, FoundX, FoundY, 336, 308, 521, 376, *20 C:\Some_Button.png
CenterImgSrchCoords("*20 C:\Some_Button.png", FoundX, FoundY)
```

**Comment by Pulover** [comment]
Will be fixed in the next release.

**Comment by Pulover** [comment]
Fixed in v5.1.2.
